docs/1. Installer le projet en local.md

```diff
@@ -23,7 +23,7 @@ Les services/outils suivants sont nécessaires pour développer :
 
 ou
 
-- PHP 7.0+
+- PHP 7.1+
```

Comment:
Ok but maybe it's not a good idea to develop on 7.1 if we are requiring only 7.0?

https://github.com/EnMarche/en-marche.fr/blob/master/docker/tools/Dockerfile#L1